### ISSUE TITLE
why the logits is added with the item_b

### ISSUE THREAD

**Issue by LeoCai** [stellaxu/DHAN#3]
![image](https://user-images.githubusercontent.com/7079275/87896406-b9306980-ca7a-11ea-98aa-948a056d6fcb.png)

![image](https://user-images.githubusercontent.com/7079275/87896399-b2095b80-ca7a-11ea-941d-2a5f2064ce92.png)

the logits is added with the item_b, what is the purpose?

thank you ~
